kmos_tools/star_offsets.py

```diff
@@ -248,9 +252,11 @@ def star_psf(exposure, clobber=True, vb=False):
     """
 
     exposure.invert = False
-
-    if clobber is True and ~os.path.exists(exposure.starfile):
-        exposure.starfile, exposure.invert = exposure.star_fit_profile()
+    #print('0')
+    #if clobber is True and ~os.path.exists(exposure.starfile): - problem with this line
+    #print('1')
+    exposure.starfile, exposure.invert = star_fit_profile(exposure) #changed this line - wasn't calling function
```

Comment:
This is fine for now. I had set it up so that if the starfile already existed you wouldn't have to make a new one, but that probably doesn't happen often. (I think it was failing as the exposure.starfile=None)